cmake/SfizzConfig.cmake

```diff
@@ -1,3 +1,10 @@
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND (CMAKE_CXX_STANDARD LESS 14 OR NOT CMAKE_CXX_STANDARD))
+    # There is a strange segfault in clang in c++11 when instantiating the
+    # envelopes in FloatEnvelopes.cpp.
+    message("Forcing C++14 to bypass a clang segfault")
```

Comment:
upgrade the xcode version instead?
(and keep back the deployment target version)

Reply:
We can try but I'm not sure. It also crashes the default clang on my machine (ubuntu 19.10). It's a really weird bug.

Reply:
And with LTO disabled?

Reply:
It fails at compile time so I guess LTO is not the issue here. I will start by trying to actually downgrade XCode to use the default Travis one and we'll see what happens :)

Reply:
Nope, either downgrading or upgrading... I guess I should file a compiler bug report :/

Reply:
Yes you should, but also, try replacing.
```
    std::function<Type(Type)> function { [](Type input) { return input; } };
```
with
```
    std::function<Type(Type)> function { [](Type input) -> Type { return input; } };
```

Reply:
Holy shit, nice catch lol